File: AtCoder/363B-Japanese_Cursed_Doll.java
import java.io.PrintWriter;
import java.util.Scanner;

public class Main {

  private static final Scanner in = new Scanner(System.in);
  private static final PrintWriter out = new PrintWriter(System.out);

  public static void main(String[] args) {
    int n = in.nextInt();
    int t = in.nextInt();
    int p = in.nextInt();

    int[] arr = new int[n];
    for (int i = 0; i < n; i++) {
      arr[i] = in.nextInt();
    }

    int count = 0;
    while (countVal(arr, t) < p) {
      addOne(arr, n);
      count++;
    }

    out.println(count);
    in.close();
    out.close();
  }

  static void addOne(int[] arr, int n) {
    for (int i = 0; i < n; i++) arr[i]++;
  }

  static int countVal(int[] arr, int t) {
    int count = 0;
    for (int val : arr) if (val >= t) count++;
    return count;
  }
}

File: AtCoder/363B-Japanese_Cursed_Doll.c
#include <stdio.h>

int countVal(int arr[], int n, int t) {
  int count = 0;
  for (int i = 0; i < n; i++)
    if (arr[i] >= t)
      count++;
  return count;
}

void plusOne(int arr[], int n) {
  for (int i = 0; i < n; i++)
    arr[i]++;
}

int main() {
  int n, t, p;
  scanf("%d %d %d", &n, &t, &p);

  int arr[n];
  for (int i = 0; i < n; i++) {
    scanf("%d", &arr[i]);
  }

  int count = 0;
  while (countVal(arr, n, t) < p) {
    plusOne(arr, n);
    count++;
  }

  printf("%d\n", count);
}
